src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -123,8 +130,8 @@ int main(int argc, const char *argv[]) {
   auto outputBinaryFlag = emitBinary ? Flags::Binary : Flags::Text;
   Output output(outfile, outputBinaryFlag, Flags::Release);
   ModuleWriter writer;
-  // writer.setDebug(options.debug);
-  writer.setDebugInfo(true);
+  writer.setDebug(options.debug);
+  writer.setDebugInfo(debugInfo);
   // writer.setDebugInfo(options.passOptions.debugInfo);
   // writer.setSymbolMap(symbolMap);
```

Comment:
Probably need to implement this line for full s2wasm parity

Reply:
We can do that as a followup maybe?

Reply:
Agree, hence lgtm